build/KoreBuild.ps1

```diff
@@ -98,38 +98,18 @@ if (!($env:Path.Split(';') -icontains $dotnetLocalInstallFolder))
     $env:Path = "$newPath"
 }
 
-# wokaround for CLI issue: https://github.com/dotnet/cli/issues/2143
-$sharedPath = (Join-Path (Split-Path ((get-command dotnet.exe).Path) -Parent) "shared");
-(Get-ChildItem $sharedPath -Recurse *dotnet.exe) | %{ $_.FullName } | Remove-Item;
-
-# We still nuget because dotnet doesn't have support for pushing packages
-$nugetExePath = Join-Path $PSScriptRoot 'nuget.exe'
-if (!(Test-Path $nugetExePath))
-{
-    Invoke-WebRequest "https://dist.nuget.org/win-x86-commandline/v4.0.0-rc4/NuGet.exe" -OutFile "$PSScriptRoot/nuget.exe"
-}
-
 $makeFileProj = "$PSScriptRoot/KoreBuild.proj"
 $msbuildArtifactsDir = "$repoFolder/artifacts/msbuild"
 $msbuildLogFilePath = "$msbuildArtifactsDir/msbuild.log"
 $msBuildResponseFile = "$msbuildArtifactsDir/msbuild.rsp"
 
-$preflightClpOption='/clp:DisableConsoleColor'
-$msbuildClpOption='/clp:DisableConsoleColor;Summary'
-if ("${env:CI}${env:APPVEYOR}${env:TEAMCITY_VERSION}${env:TRAVIS}" -eq "")
-{
-    # Not on any of the CI machines. Fine to use colors.
-    $preflightClpOption=''
-    $msbuildClpOption='/clp:Summary'
-}
-
```

Comment:
>  There's no point to colour in the CI builds.

I disagree. Color is useful, esp. when there are warnings and errors. AppVeyor is the only one that strips color. TeamCity shows color when you view logs in "console view".
![image](https://cloud.githubusercontent.com/assets/2696087/25498317/6138ccee-2b3c-11e7-8ab7-da28fe45423a.png)